tests/integration/post_training/test_post_training.py

```diff
@@ -19,10 +18,11 @@
     LoraFinetuningConfig,
     TrainingConfig,
 )
+from llama_stack.log import get_logger
 
 # Configure logging
-logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s", force=True)
-logger = logging.getLogger(__name__)
+logger = get_logger(name=__name__, category="post_training")
```

Comment:
hmmm is this integration or post_training?

Reply:
lgtm 👍🏽